.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+python: 2.7
+
```

Comment:
Since we don't need sudo, we can set
`sudo: false`
and use the new Docker infrastructure (which is way faster).

More info: http://cdunn2001.blogspot.com/2015/06/using-c11-in-travis-ci.html (this also show how to install packages in Ubuntu without using sudo)